python/apps/taiga/src/taiga/stories/attachments/api.py

```diff
@@ -0,0 +1,83 @@
+# -*- coding: utf-8 -*-
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+#
+# Copyright (c) 2023-present Kaleidos INC
+
+from functools import partial
+
+from taiga.attachments import services as attachments_services
+from taiga.attachments.models import Attachment
+from taiga.attachments.serializers import AttachmentSerializer
+from taiga.base.api import AuthRequest
+from taiga.base.api.permissions import check_permissions
+from taiga.base.validators import B64UUID, UploadFile
+from taiga.exceptions.api.errors import ERROR_403, ERROR_404, ERROR_422
+from taiga.permissions import HasPerm
+from taiga.routers import routes
+from taiga.stories.attachments import events
+from taiga.stories.stories.api import get_story_or_404
+
+# PERMISSIONS
+CREATE_STORY_ATTACHMENT = HasPerm("modify_story") | HasPerm("comment_story")
+LIST_STORY_ATTACHMENTS = HasPerm("view_story")
+
+################################################
+# create story attachment
+################################################
+
+
+@routes.story_attachments.post(
+    "/projects/{project_id}/stories/{ref}/attachments",
+    name="project.story.attachments.create",
+    summary="Attach a file to an story",
+    responses=ERROR_403 | ERROR_404 | ERROR_422,
+    response_model=AttachmentSerializer,
+)
+async def create_story_attachments(
```

Comment:
If I understand this well, the user can create one attachment at a time. Then it would be `create_story_attachment`.

Reply:
fixed